### PR TITLE
Style resource cards on explore all page

### PR DESCRIPTION
**Pull request by evanschris**

### CODE REVIEW

File: app/templates/ai_lab/_partials/resource.html [hunk 1]
```diff
@@ -1,14 +1,53 @@
...
+            "
+          alt="{{ alt_text }}"
+        />
+      {% else %}
+        <div class="nhsuk-promo__img nhsai__resource_without_image"></div>
+      {% endif %}
+      <div class="nhsuk-promo__content">
+        <p class="nhsai_resouce__type">{{ resource|resource_type }}</p>
+        <!-- link to be used later (possibly not on this page?) 
+          <a href="{{ page|link_to_resource_type:page }}">{{ page|resource_type }}</a> -->
```
Comment (pezholio) [inline app/templates/ai_lab/_partials/resource.html]: Tiny thing, but these should have the `resource` piped in, not the `page`, so:

```
<a href="{{ resource|link_to_resource_type:page }}">{{ resource|resource_type }}</a>
```

File: app/templates/ai_lab/_partials/resource.html [hunk 2]
```diff
@@ -1,14 +1,53 @@
...
+        <h3 class="nhsuk-promo__heading">{{ resource.title }}</h3>
+        <p class="nhsuk-promo__description">{{ resource.summary }}</p>
+        <div class="nhsai_resouce__tags">
+          {% for topic in resource.topics.all %}
+            <span class="nhsai_resouce__tag">{{ topic }}</span>
```
Comment (pezholio) [inline app/templates/ai_lab/_partials/resource.html]: Do we need to set a `word-wrap` or something on the tags? See:

image

File: app/assets/_dev/scss/ailab/_ai_lab_resources.scss [hunk 3]
```diff
@@ -45,4 +45,53 @@
             padding-left: 17px;
         }
     }
+}
+
+.nhsai-resources-group {
+    .nhsuk-promo-group__item {
+        margin-bottom: 32px;
+        .nhsai_resouce__category {
```
Comment (pezholio) [inline app/assets/_dev/scss/ailab/_ai_lab_resources.scss]: Typo on `resource` here (and a few other places)

File: app/assets/_dev/scss/ailab/_ai_lab_resources.scss [hunk 4]
```diff
@@ -45,4 +45,53 @@
...
+            opacity: 0.9;
+        }
+        .nhsuk-promo__img {
+            border-bottom: 5px solid $nhsx-blue;
+        }
+        .nhsai__resource_without_image {
+            min-height: 196px;
+            background: $nhs-mid-grey;
+        }
+        .nhsai_resouce__type {
```
Comment (pezholio) [inline app/assets/_dev/scss/ailab/_ai_lab_resources.scss]: Typo 😺

File: app/assets/_dev/scss/ailab/_ai_lab_resources.scss [hunk 5]
```diff
@@ -45,4 +45,53 @@
...
+            margin-bottom: 0;
+            text-transform: uppercase;
+            font-weight: 600;
+            font-size: 1rem;
+        }
+        .nhsai_resouce__tags {
```
Comment (pezholio) [inline app/assets/_dev/scss/ailab/_ai_lab_resources.scss]: Here too 👍

File: app/assets/_dev/scss/ailab/_ai_lab_resources.scss [hunk 6]
```diff
@@ -45,4 +45,53 @@
...
+            .nhsai_resouce__tag {
```
Comment (pezholio) [inline app/assets/_dev/scss/ailab/_ai_lab_resources.scss]: Another

File: app/assets/_dev/scss/ailab/_ai_lab_resources.scss [hunk 7]
```diff
@@ -45,4 +45,53 @@
...
+                padding: 5px 10px;
+                margin-right: 5px;
+            }
+        }
+        .nhsuk-promo__link-wrapper {
+            border: none;
+            position: relative;
+
+            &:hover {
+                .nhsai_resouce__category {
```
Comment (pezholio) [inline app/assets/_dev/scss/ailab/_ai_lab_resources.scss]: Another 😃

File: app/templates/ai_lab/_partials/resource.html [hunk 8]
```diff
@@ -1,14 +1,53 @@
...
-    <div class="nhsuk-promo__content">
-      <span>
-        <a href="{{ resource|link_to_resource_type:page }}">{{ resource|resource_type }}</a>
-      </span>
-      <a href="{{ resource.url }}">
-        <h2>{{ resource.title }}</h2>
-      </a>
-    </div>
+    <a href="{{ resource.url }}" class="nhsuk-promo__link-wrapper">
+      <span class="nhsai_resouce__category">{{ resource.get_parent }}</span>
```
Comment (pezholio) [inline app/templates/ai_lab/_partials/resource.html]: Typo here

File: app/templates/ai_lab/_partials/resource.html [hunk 9]
```diff
@@ -1,14 +1,53 @@
...
+              {{ six_image.url }} 1125w,
+              {{ seven_image.url }} 1534w,
+            "
+          alt="{{ alt_text }}"
+        />
+      {% else %}
+        <div class="nhsuk-promo__img nhsai__resource_without_image"></div>
+      {% endif %}
+      <div class="nhsuk-promo__content">
+        <p class="nhsai_resouce__type">{{ resource|resource_type }}</p>
```
Comment (pezholio) [inline app/templates/ai_lab/_partials/resource.html]: Another typo

File: app/templates/ai_lab/_partials/resource.html [hunk 10]
```diff
@@ -1,14 +1,53 @@
...
+        <!-- link to be used later (possibly not on this page?) 
+          <a href="{{ page|link_to_resource_type:page }}">{{ page|resource_type }}</a> -->
+        <h3 class="nhsuk-promo__heading">{{ resource.title }}</h3>
+        <p class="nhsuk-promo__description">{{ resource.summary }}</p>
+        <div class="nhsai_resouce__tags">
```
Comment (pezholio) [inline app/templates/ai_lab/_partials/resource.html]: Probably best to do a find and replace on these 😆

**Review by pezholio — CHANGES_REQUESTED** [review]
Looks banging, just a couple of tiny points (and a few typos - sorry!)

File: app/templates/ai_lab/_partials/resource.html (hunk 1 above)
Comment (pezholio) [reply app/templates/ai_lab/_partials/resource.html]: (I know this is commented out for now, but it just saves confusion later on 😄 )

File: app/templates/ai_lab/_partials/resource.html (hunk 2 above)
Comment (evanschris) [reply app/templates/ai_lab/_partials/resource.html]: Good point my example tags were not long enough 😂 

Its the very confusingly named - `white-space: nowrap;`

**Pull request merged**